PowerShell/BuildingBlocks/Authentication/GetTicket.ps1

```diff
@@ -0,0 +1,5 @@
+﻿# This script shows how to get an authentication ticket
+
+# Get a ticket
```

Comment:
Question: For retrieving a MigrationWiz ticket, you use the comment "Get a ticket for MigrationWIz". Does that mean that when we retrieve a BT Ticket, we just want to associate the notion of a generic ticket to it? I see that you wrote the comment "Get a Ticket" for retrieving a BT ticket.

Reply:
Yes, that's because we use $ticket and $mwTicket in the sample scripts for a BitTitan ticket and a MigrationWiz ticket respectively.
The idea is that MigrationWiz is a sub-product of BitTitan, that's why it's more a specific case. It's not like we have two products: BitTitan vs MigrationWiz.